lib/open_api_spex/schema.ex

```diff
@@ -250,6 +251,7 @@ defmodule OpenApiSpex.Schema do
           xml: Xml.t() | nil,
           externalDocs: ExternalDocumentation.t() | nil,
           example: any,
+          example: [any],
```

Comment:
This should be `examples`

Reply:
Thanks, fixed!

(you'd think with such a small diff I would avoid such silly mistakes)